scene/main/node.cpp

```diff
@@ -3011,6 +3011,11 @@ void Node::get_argument_options(const StringName &p_function, int p_idx, List<St
 	String pf = p_function;
 	if ((pf == "has_node" || pf == "get_node") && p_idx == 0) {
 		_add_nodes_to_options(this, this, r_options);
+	} else if ((pf == "add_to_group" || pf == "remove_from_group" || pf == "is_in_group") && p_idx == 0) {
```

Comment:
```suggestion
	} else if (p_idx == 0 && (pf == "add_to_group" || pf == "remove_from_group" || pf == "is_in_group")) {
```
I'd move the cheaper check first